src/python/ensembl/production/xrefs/Base.py

```diff
@@ -838,11 +836,11 @@ def get_xref_mapper(self, xref_url: str, species: str, base_path: str, release:
     pep_path = self.get_path(base_path, species, release, 'ensembl', 'peptides.fa');
 
     # Try to find a species-specific mapper first
-    module_name = f'ensembl.xrefs.mapper.{species}'
+    module_name = f'ensembl.xrefs.mappers.{species}'
```

Comment:
This shouldn't be `ensembl.production.xrefs.mappers.BasicMapper` (with `production`)?

Reply:
mappers module is missing in ensmebl-production repo it exists in a separate repo (https://github.com/Ensembl/ensembl_nf/tree/main/src/python/ensembl/production/xrefs/mappers) 
yes, you're right It is in the production namespace. we need to check with @TamaraNaboulsi on this 
 we will fix this in the release/114 branch

Reply:
You're right John, this should be `ensembl.production.xrefs.mappers` but the mappers modules haven't been added yet as those belong to the second pipeline that hasn't yet been merged. I included this method (get_xref_mapper) by mistake. Will remove it in my upcoming PR for release/114.